src/store/searchFilters.js

```diff
@@ -15,13 +15,13 @@ export const mutations = {
     setFilterButtons(state) {
         state.filterButtons.push([
             {
-                "title": "Match all terms",
+                "title": "Match all term",
```

Comment:
If im right, the correct way is:
- match all terms (plural)
- match any term (singular)

Reply:
@terazus has asked for the correctness of the spelling and as I asked @knirirr about it. Appreciate to let us know.

Reply:
I asked @knirirr and he said my proposal is right.

Reply:
fixed